java/ruleXtract/src/main/java/uk/ac/cam/eng/extraction/hadoop/util/CopyRecordsForTesting.java

```diff
@@ -0,0 +1,67 @@
+/*******************************************************************************
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use these files except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *******************************************************************************/
+
+package uk.ac.cam.eng.extraction.hadoop.util;
+
+import java.io.IOException;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.io.SequenceFile;
+import org.apache.hadoop.io.SequenceFile.CompressionType;
+import org.apache.hadoop.io.Writable;
+import org.apache.hadoop.util.ReflectionUtils;
+
+/**
+ * Copy records from a sequence file for testing
+ * 
+ * @author aaw35
+ *
+ */
+
+public class CopyRecordsForTesting {
+
+	public static void main(String[] args) throws IOException {
+		if (args.length != 3) {
+			System.err
```

Comment:
I'm using the standard eclipse indention style, which encourages narrow lines. I don't really want to mess around with custom eclipse styles, and would encourage other people to use the standard eclipse styles.